backend/business-partner-agent/src/main/java/org/hyperledger/bpa/api/aries/SchemaAPI.java

```diff
@@ -55,13 +55,17 @@
     private List<CredDef> credentialDefinitions;
 
     public static SchemaAPI from(BPASchema s) {
+        return from(s, true, true);
```

Comment:
can you leave a comment on this? i assume it's temporary?

Reply:
That one is in use elsewhere in the application and uses the previous behaviour - so it stays; the override (with booleans) is used when I list credential definitions, so I don't need the schema's list of cred defs. This is permanent, but only until Phillip moves to a completely different mapping tool.